apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -163,9 +163,13 @@ export default function ScheduleCalendar(props: ScheduleCalendarProps) {
     const calendarTimeFormat = isMilitaryTime ? 'HH:mm' : 'h:mm A';
     const calendarGutterTimeFormat = isMilitaryTime ? 'HH:mm' : 'h A';
 
-    const defaultFinals = getDefaultFinalsStartDate();
-    const finalsDateFormat = defaultFinals ? 'ddd MM/DD' : 'ddd';
-    const date = showFinalsSchedule && defaultFinals ? defaultFinals : new Date(2018, 0, 1);
+    const finalsDate =
+        eventsInCalendar.length > 0 ? getFinalsStartDateForTerm(eventsInCalendar[0].term) : getDefaultFinalsStartDate();
```

Comment:
The first event in the calendar isn't guaranteed to have a term (custom events), and will raise an error.